brevo/assets/logs/brevo_tests.yaml

```diff
@@ -0,0 +1,1069 @@
+id: brevo
+tests:
+  - sample: |-
```

Comment:
In some sample you provide timestamp, in some other you simply provide this `ts` property that seems use for the date. Is that intended, or should we always provide the timestamp property?

Please note, we do handle only the timesamp in miliseconds in our systems. You can find more details [here](https://docs.datadoghq.com/service_management/events/pipelines_and_processors/date_remapper/).

So 2 solutions:
- either always provide timestamp and update the sample here
- add remapping for `ts` property as date remapper transforming by multiplying it by 1000 as it is a timstamp in seconds

Reply:
@thibaultkrebs We have modified the samples with `timestamp` property.

Reply:
I have seen still some samples without this property, please make sure all of them are properly set